src/TableOfContents/index.tsx

```diff
@@ -214,6 +214,7 @@ export function TableOfContents<T extends TableOfContentsItem = TableOfContentsI
             className={cn(`m-${padding}`, { 'mr-0': hasFilter })}
           >
             <Button
+              className="w-48 truncate"
```

Comment:
I suggest a max width, rather than width. If we don't have appropriate max width tailwind classes, can always just fall back to style if needed.

Reply:
Yeah, we don't. Alright, style it is. This makes the UI less jittery though with changing lengths of org names.